### PR TITLE
[search] Separate PreRanker::Params::m_minDistanceOnMapBetweenResults…

### PR DESCRIPTION
**Pull request by tatiana-yan**
… and Ranker::Params::m_minDistanceBetweenResultsM

### CODE REVIEW

**Comment by tatiana-yan** [comment]
JTTP

File: search/search_integration_tests/interactive_search_test.cpp [hunk 1]
```diff
@@ -126,38 +126,36 @@ UNIT_CLASS_TEST(InteractiveSearchTest, Smoke)
 
 UNIT_CLASS_TEST(InteractiveSearchTest, NearbyFeaturesInViewport)
 {
-  double const kEps = 1e-5;
+  double const kEps = 0.1;
```
Comment (mpimenov) [inline search/search_integration_tests/interactive_search_test.cpp]: s/kEps/kSide/ ?

Comment (mpimenov) [reply search/search_integration_tests/interactive_search_test.cpp]: Ah, it's not a square anymore. Ok.

File: search/pre_ranker.hpp [hunk 2]
```diff
@@ -28,7 +28,7 @@ class PreRanker
   {
     m2::RectD m_viewport;
 
-    // A minimum distance between search results in meters, needed for
+    // A minimum distance between search results in mercators, needed for
```
Comment (mpimenov) [inline search/pre_ranker.hpp]: s/A minimum/Minimal/g ?

Comment (tatiana-yan) [reply search/pre_ranker.hpp]: done

**Pull request merged**